src/shogun/lib/config.h.in

```diff
@@ -95,8 +95,6 @@
 #cmakedefine HAVE_ALIGNED_MALLOC 1
 #cmakedefine HAVE_POSIX_MEMALIGN 1
 
-#cmakedefine HAVE_CXX11 1
```

Comment:
have you removed all HAVE_CXX11 macros? otherwise this will break other things

Reply:
Yes. I have removed all the HAVE_CXX11 macros including the gpl submodule.